src/microbe_stage/MicrobeStage.cs

```diff
@@ -384,10 +384,6 @@ public override void _Process(float delta)
             }
         }
 
-        // Start auto-evo if not already and settings have auto-evo be started during gameplay
-        if (Settings.Instance.RunAutoEvoDuringGamePlay)
```

Comment:
Could you keep the auto evo start here?
Because that way it won't trigger on the same frame that saving has *started* instead it will naturally trigger on the frame that saving has finished.